apps/angular/signal-input/src/app/user.component.ts

```diff
@@ -19,23 +20,18 @@ const ageToCategory = (age: number): Category => {
   standalone: true,
   imports: [TitleCasePipe],
   template: `
-    {{ fullName | titlecase }} plays tennis in the {{ category }} category!!
+    {{ fullName() | titlecase }} plays tennis in the {{ category() }} category!!
   `,
   host: {
     class: 'text-xl text-green-800',
   },
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class UserComponent implements OnChanges {
-  @Input({ required: true }) name!: string;
-  @Input() lastName?: string;
-  @Input() age?: string;
+export class UserComponent {
+  name = input.required<string>();
+  lastName = input<string>();
+  age = input<number, string>(0, { transform: numberAttribute });
```

Comment:
When the type is inferred, you don't need to specific it.

Reply:
there you go [d455869](https://github.com/tomalaforge/angular-challenges/pull/511/commits/d455869b601c3f31e4252f86ed2610ff2507e638)

do you see anything else I could do better ?

Reply:
I added one last comment since you asked 😅